services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java

```diff
@@ -209,9 +209,13 @@ protected Response createResponse(LoginFormsPages page) {
                 BrokeredIdentityContext brokerContext = (BrokeredIdentityContext) this.attributes.get(IDENTITY_PROVIDER_BROKER_CONTEXT);
                 String idpAlias = brokerContext.getIdpConfig().getAlias();
                 idpAlias = ObjectUtil.capitalize(idpAlias);
+                String displayName = idpAlias;
+                if (!ObjectUtil.isBlank(brokerContext.getIdpConfig().getDisplayName())) {
+                    displayName = brokerContext.getIdpConfig().getDisplayName();
+                }
 
                 attributes.put("brokerContext", brokerContext);
-                attributes.put("idpAlias", idpAlias);
+                attributes.put("idpDisplayName", displayName);
```

Comment:
`idpAlias` must be retained, `idpDisplayName` is a new attribute

Reply:
Hi @hmlnarik, please check again